pkgs/applications/editors/vim/plugins/overrides.nix

```
@@ -2890,6 +2890,10 @@ in
 
   sniprun = callPackage ./non-generated/sniprun { };
 
+  sonarlint-nvim = super.sonarlint-nvim.overrideAttrs {
+    dependencies = [nodejs];
```

```suggestion
    runtimeDeps = [nodejs];
```

`dependencies` is only for plugins (also `runtimeDeps` is used only in neovim in unstable)